addons/fluxcd/resources/components/image-automation-controller.cue

```diff
@@ -0,0 +1,71 @@
+output: {
```

Comment:
This is not a recommend way for now. You can announce a cue block here , then referenced by `template.cue`. Please refer to :https://github.com/kubevela/catalog/pull/424/files